src/app/state/transaction/saga.ts

```diff
@@ -7,7 +7,7 @@ import { call, put, race, select, take, takeEvery } from 'typed-redux-saga'
 import { ErrorPayload, ExhaustedTypeError, WalletError, WalletErrors } from 'types/errors'
 
 import { transactionActions } from '.'
-import { sign } from '../ledger/saga'
+import { sign } from '../importaccounts/saga'
```

Comment:
I didn't notice before, this ledger-specific sign function is now oddly named / oddly placed

Reply:
indeed, should sign and getUSBTransport generator functions be moved to `app/lib/ledger.ts` ?

Reply:
yep

maybe also have fewer ledger sign functions?
- sign from ../importaccounts/saga calls
- OasisTransaction.signUsingLedger calls
- LedgerSigner.sign calls
- OasisApp.sign